### ISSUE TITLE
No results are obtained in new facebook page template

### ISSUE THREAD

**Issue by gerickt** [shaikhsajid1111/facebook_page_scraper#12]
Hello, in pages that have migrated to the new template, it is not possible to recover the posts.
In pages that keep the old template, it works without problem.

Do you plan to support the new facebook template in the future?

**Comment by shaikhsajid1111** [comment]
Yeah, I do plan to support but currently, I'm not getting that new UI in my country and the same goes for many different countries.
